src/ai-bundle/config/options.php

```diff
@@ -96,6 +96,7 @@
                                 ->end()
                             ->end()
                         ->end()
+                        ->booleanNode('track_token_usage')->defaultFalse()->end()
```

Comment:
please add `->info()` node and add something to the README about this new option, thanks

Reply:
Done.
Also added some info in `ai-bundle/doc/index.rst`.